packages/app/pages/Api/agent.tsx

```diff
@@ -0,0 +1,145 @@
+/**
+ * This exports a React element which displays the current users agent, a list of all agents,
+ * and provides a section to select a single agent out of that list
+ *
+ * @package: REA app
+ * @author:  Steven Fontaine <fontainesw@msoe.edu>
+ * @since:   2018-01-25
+ */
+
+import * as React from "react";
+import "./api.css"
+import getMyAgent from "../../../ui-bindings/Agent/getMyAgent.js";
+import getAllAgents from "../../../ui-bindings/Agent/getAllAgents.js";
+import getAgentById from "../../../ui-bindings/Agent/getAgentById";
+
+export const Agent = (props) => {
```

Comment:
It might be worth reassigning the agent variable. (ex: let agent = props.agent) to make it easier to read

Reply:
I agree, I will make that change

Reply:
Resolved